content/o11y/openshift-logging-lokistack/index.md

```diff
@@ -0,0 +1,572 @@
+---
+date: '2021-07-27'
+title: Configuring OpenShift Logging using LokiStack on ROSA and (soon) ARO
+tags: ["Observability", "OCP"]
+authors:
+  - Andy Repton
+---
+
+A guide to shipping logs and metrics on OpenShift using the new LokiStack setup
+
+## Overview of the components of OpenShift Cluster Logging
+
+![logging overview](./logging-overview.png)
+
+## Prerequisites
+
+1. OpenShift CLI (oc)
+1. Rights to install operators on the cluster
+1. Access to create S3 buckets (AWS/ROSA), Blob Storage Container (Azure), Storage Bucket (GCP)
+
+## For ROSA
```

Comment:
I think the other Cloud Experts tutorials name this section "Setting up your environment", so I'd suggest that here as well.